cpp/dolfinx/mesh/Geometry.cpp

```diff
@@ -75,6 +77,11 @@ const std::vector<std::int64_t>& Geometry::global_indices() const
   return _global_indices;
 }
 //-----------------------------------------------------------------------------
+const std::vector<std::int64_t>& Geometry::flags() const
```

Comment:
`mesh::Geometry` already has `std::vector<std::int64_t> _global_indices;` for this purpose.